extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/AbstractGuacamoleTunnelService.java

```diff
@@ -392,7 +395,29 @@ private GuacamoleSocket getUnconfiguredGuacamoleSocket(
         public ConnectionCleanupTask(ActiveConnectionRecord activeConnection) {
             this.activeConnection = activeConnection;
         }
-        
+
+
+
+        /***
+         * @Author zero
+         * @Description 将活动会话记录里的 会话开始时间变更为GUAC_DATE,GUAC_TIME
+         * @InitDate 10:53 2019/10/12
+         * @Param [activeConnection, tokens]
+         **/
```

Comment:
The javadoc comments should follow the format of the other comments throughout the code.  Please be consistent with the style used in the code.

Reply:
Documentation will also need to be in English.